cypress/integration/Services/Measure Service/MeasureBundle.spec.ts

```diff
@@ -0,0 +1,383 @@
+                    headers: {
+                        authorization: 'Bearer ' + accessToken.value
+                    }
+                }).then((response) => {
+                    expect(response.status).to.eql(200)
+                    expect(response.body.resourceType).to.eql('Bundle')
+                    expect(response.body.entry).to.be.a('array')
+                    expect(response.body.entry[0].resource.resourceType).to.eql('Measure')
+                    expect(response.body.entry[0].resource.effectivePeriod).to.have.property('start')
+                    expect(response.body.entry[0].resource.effectivePeriod).to.have.property('end')
+                    expect(response.body.entry[0].resource.library[0]).is.not.empty
+                    expect(response.body.entry[0].resource.group[0].population[0].code.coding[0].code).to.eql('initial-population')
+                    expect(response.body.entry[0].resource.group[0].population[0].criteria.expression).to.eql('SDE Payer')
+                    expect(response.body.entry[0].resource.group[0].population[1].code.coding[0].code).to.eql('denominator')
+                    expect(response.body.entry[0].resource.group[0].population[1].criteria.expression).to.eql('SDE Sex')
+                    expect(response.body.entry[0].resource.group[0].population[2].code.coding[0].code).to.eql('denominator-exclusion')
+                    expect(response.body.entry[0].resource.group[0].population[2].criteria.expression).to.eql('Absence of Cervix')
+                    expect(response.body.entry[0].resource.group[0].population[3].code.coding[0].code).to.eql('denominator-exception')
+                    expect(response.body.entry[0].resource.group[0].population[3].criteria.expression).to.eql('SDE Ethnicity')
+                    expect(response.body.entry[0].resource.group[0].population[4].code.coding[0].code).to.eql('numerator')
+                    expect(response.body.entry[0].resource.group[0].population[4].criteria.expression).to.eql('SDE Race')
+                    expect(response.body.entry[0].resource.group[0].population[5].code.coding[0].code).to.eql('numerator-exclusion')
+                    expect(response.body.entry[0].resource.group[0].population[5].criteria.expression).to.eql('Surgical Absence of Cervix')
+                    expect(response.body.entry[1].resource.resourceType).to.eql('Library')
+                })
+            })
+        })
+    })
+})
+/* describe.only('Measure Bundle end point returns nothing with Measure CQL missing FHIRHelpers include line', () => {
+    before('Create Measure',() => {
+
+        cy.setAccessTokenCookie()
+
+        //Create New Measure
+        cy.getCookie('accessToken').then((accessToken) => {
+            cy.request({
+                url: '/api/measure',
+                headers: {
+                    authorization: 'Bearer ' + accessToken.value
+                },
+                method: 'POST',
+                body: {
```

Comment:
Still need this commented out code?

Reply:
you can add it.skip, might be a bit cleaner. Then just put a comment above that about he open bug and jira ticket

Reply:
.skip has been added